radixdlt/src/main/java/com/radixdlt/consensus/safety/VertexHasher.java

```diff
@@ -1,33 +1,25 @@
-/*
- * (C) Copyright 2020 Radix DLT Ltd
- *
- * Radix DLT Ltd licenses this file to you under the Apache License,
- * Version 2.0 (the "License"); you may not use this file except in
- * compliance with the License.  You may obtain a copy of the
- * License at
- *
- *  http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
- * either express or implied.  See the License for the specific
- * language governing permissions and limitations under the License.
- */
-
-package org.radix.serialization;
+/*
+ * (C) Copyright 2020 Radix DLT Ltd
+ *
+ * Radix DLT Ltd licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied.  See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
 
-import com.radixdlt.consensus.NewRound;
-import com.radixdlt.consensus.Round;
-import com.radixdlt.consensus.messages.NewRoundMessage;
+package com.radixdlt.consensus.safety;
 
-public class NewRoundMessageSerializeTest extends SerializeMessageObject<NewRoundMessage> {
-	public NewRoundMessageSerializeTest() {
-		super(NewRoundMessage.class, NewRoundMessageSerializeTest::get);
-	}
+import com.radixdlt.consensus.VertexMetadata;
+import com.radixdlt.crypto.Hash;
 
-	private static NewRoundMessage get() {
-		NewRound testRound = new NewRound(Round.of(1234567890L));
-		return new NewRoundMessage(1234, testRound);
-	}
+public interface VertexHasher {
+	Hash hash(VertexMetadata vertexMetadata);

```

Comment:
As noted there are now two hashes per Vertex, one a hash of Vertex and another a hash of VertexMetadata. Need to discuss this...

Reply:
Yeah, that's definitely weird. I want to unify them as that's just confusing.

Reply:
Resolved by removing VertexHasher and using Vertex.getId directly for now.